low-level/files/opt/seagate/sspl/setup/manifest_support_bundle

```diff
@@ -83,21 +83,25 @@ class ManifestSupportBundle(object):
             except (FileNotFoundError, OSError) as e:
                 print(f"Unable to get lr serial number from the given path : {self.lr_serial_number_path}")
                 lr_serial_number = "NA"
-
             try:
+                self.support_common_data = {
+                    "cluster_id": self.cluster_id,
+                    "site_id": self.site_id,
+                    "rack_id": self.rack_id,
+                    "node_id": self.node_id
+                }
+                self.capture_node_data()
                 sspl_state=open(f"/var/{PRODUCT_FAMILY}/sspl/data/state.txt").readline().rstrip().split("=")
                 # Capturing enclosure data only when sspl state is active
                 sspl_state = sspl_state[1] if len(sspl_state)==2 else ''
                 if sspl_state == 'active':
-                    subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-e", "-s"], shell=False)
-                else:
-                    print("SSPL state is not active, So enclosure data are ignored to collecte.")
-                    subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-s"], shell=False)
+                    if self.storage_type == self.STORAGE_TYPE_RBOD:
+                        self.capture_enclosure_data()
             except (FileNotFoundError, OSError) as e:
                 print(f"Failed to open the SSPL 'state.txt' file with an error '{e}',\
                     Can't determine the SSPL state, So enclosure data also being collected.")
                 sspl_state == 'active'
-                subprocess.call([f"{RESOURCE_HEALTH_VIEW}", "-n", "-e", "-s"], shell=False)
+                self.capture_enclosure_data()
```

Comment:
On the above line (103). can you correct it to sspl_state = 'active' instead of '=='

Reply:
Done